src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -18,14 +18,15 @@ var ReactGenericBatching = require('ReactGenericBatching');
 var emptyObject = require('emptyObject');
 
 import type { TestRendererOptions } from 'ReactTestMount';
+import type { ReactText } from 'ReactTypes';
 
 type ReactTestRendererJSON = {|
   type : string,
-  props : {[propName: string] : string },
+  props : {[propName: string] : any },
   children : null | Array<ReactTestRendererNode>,
   $$typeof ?: Symbol, // Optional because we add it with defineProperty().
 |};
-type ReactTestRendererNode = ReactTestRendererJSON | string;
+type ReactTestRendererNode = ReactTestRendererJSON | ReactText;
```

Comment:
I have to double-check, but I don’t think the fiber version would ever return a number since it’s always converted to strings inside fiber.